WcaOnRails/spec/controllers/competitions_controller_spec.rb

```diff
@@ -597,8 +597,8 @@
         expect(competition.results_posted_at).to be nil
         get :post_results, id: competition
         post = assigns(:post)
-        expect(post.body).to include "World records: Jeremy Fleischman 3x3 one-handed 50.00 (average), Vincent Sheu (2006SHEU01) 3x3 fewest moves 25 (single), 3x3 fewest moves 26.00 (average), Vincent Sheu (2006SHEU02) 2x2 Cube 10.00 (single)"
-        expect(post.body).to include "North American records: Jeremy Fleischman 3x3 one-handed 41.00 (single), 3x3 one-handed 40.00 (single)"
+        expect(post.body).to include "World records: Jeremy Fleischman 3x3x3 One Handed 50.00 (average), Vincent Sheu (2006SHEU01) 3x3x3 Fewest Moves 25 (single), 3x3x3 Fewest Moves 26.00 (average), Vincent Sheu (2006SHEU02) 2x2x2 Cube 10.00 (single)"
```

Comment:
So this is our longest line of Ruby code? I'd say it's time to split it up into multiple lines. I like to write multiline strings like this:

```ruby
expect(post.body).to include(
  "World records: Jeremy Fleischman 3x3x3 One Handed 50.00 (average), " +
  "Vincent Sheu (2006SHEU01) 3x3x3 Fewest Moves 25 (single), " +
  "3x3x3 Fewest Moves 26.00 (average), " +
  "Vincent Sheu (2006SHEU02) 2x2x2 Cube 10.00 (single)"
)
```